src/components/mypage/Profile.tsx

```diff
@@ -239,6 +249,7 @@ export default function Profile({ data: userData }: ProfileProps) {
             name={nickname}
             title={`${modalType === 'follower' ? '을 팔로우' : '이 팔로잉'}`}
             followUserList={followData?.list}
```

Comment:
```suggestion
            followUserList={modalType === 'follower' ? follower?.list : followee?.list}
```
이렇게 바로 전달하면 될 거같은데 상태로 데이터를 관리한 이유가 있을까요?